executers/gopackage.go

```diff
@@ -31,6 +34,7 @@ func (dependencyPackage *Package) New(cachePath string, dep Package) GoPackage {
 	dependencyPackage.modContent = dep.modContent
 	dependencyPackage.zipPath = dep.zipPath
 	dependencyPackage.version = dep.version
+	dependencyPackage.infoPath = dep.infoPath
```

Comment:
The above line is duplicated (see a few lines after).

Reply:
Sorry for that. Happened due to a merge conflict. Removed one line